packages/js/product-editor/src/components/radio-field/types.ts

```diff
@@ -3,8 +3,18 @@
  */
 import { RadioControl } from '@wordpress/components';
 
-export type RadioFieldProps< T > = Omit< RadioControl.Props< T >, 'label' > & {
+export type RadioFieldProps = Omit<
+	typeof RadioControl,
+	'label' | 'value' | 'options'
+> & {
 	title: string;
 	description?: string;
 	disabled?: boolean;
+	className?: string;
+	onChange( value: string ): void;
+	selected: string | undefined;
```

Comment:
Any particular reason you went with `| undefined` versus the optional `:?`?

Reply:
Good point. I fixed it with 074c3aa8a9fa9c7e24cdbcf33865803c912ce4c8